packages/nimble-tokens/package.json

```
@@ -3,21 +3,17 @@
   "version": "6.15.0",
   "description": "Design tokens for the NI Nimble Design System",
   "scripts": {
-    "build": "npm run build:svg-to-ts && npm run build:ts && npm run build:svg-to-ico && npm run build:generate-font-scss && npm run build:style-dictionary",
+    "build": "npm run build:svg-to-ts && npm run build:ts && npm run build:generate-font-scss && npm run build:style-dictionary",
     "build:svg-to-ts": "svg-to-ts-constants --config build/svg-to-ts-config.js",
-    "build:svg-to-ico": "node build/svg-to-ico.js",
     "build:generate-font-scss": "node build/generate-font-scss.js",
     "build:style-dictionary": "cd source/styledictionary && node build.js",
     "build:ts": "tsc -p ./tsconfig.json",
     "build:ts:watch": "tsc -p ./tsconfig.json -w",
     "format": "eslint . --fix",
     "lint": "eslint .",
-    "pack": "npm run pack:npm && npm run pack:nuget",
+    "pack": "npm run pack:npm",
     "pack:npm": "npm pack",
-    "pack:nuget": "cross-env-shell dotnet pack ./NimbleTokens --configuration Release -p:PackageVersion=$npm_package_version",
-    "invoke-publish": "npm run invoke-publish:setup && npm run invoke-publish:nuget && npm publish",
-    "invoke-publish:setup": "rimraf --glob \"NimbleTokens/bin/Release/*.nupkg\" && npm run pack",
-    "invoke-publish:nuget": "cross-env-shell dotnet nuget push \"NimbleTokens/bin/Release/*.nupkg\" -k $NUGET_SECRET_TOKEN -s \"https://api.nuget.org/v3/index.json\""
+    "invoke-publish": "npm run invoke-publish:setup && npm publish"
```

`invoke-publish:setup` was removed

```suggestion
    "invoke-publish": "npm publish"
```